apps/prairielearn/src/tests/groupExamRolePermissions.test.ts

```
@@ -426,21 +428,23 @@ describe('Assessment instance with group roles & permissions - Exam', function (
-
-    before('set up testing server', helperServer.before(TEST_COURSE_PATH));
-    before('set authenticated user', function (callback) {
+    // set up testing server
+    beforeAll(helperServer.before(TEST_COURSE_PATH));
+    // set authenticated user
+    beforeAll(function (callback) {
       storedConfig.authUid = config.authUid;
       storedConfig.authName = config.authName;
       storedConfig.authUin = config.authUin;
       callback(null);
     });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeAll duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)